DevAAC/Models/Player.php

```diff
@@ -224,6 +224,16 @@ public function houseBids()
         return $this->hasMany('DevAAC\Models\House', 'highest_bidder');
     }
 
+    public function town()
+    {
+        return $this->belongsTo('DevAAC\Models\Town');
+    }
+
+    public function getTownNameAttribute()
+    {
+        return $this->town->name;
```

Comment:
You should check if $this->town is not null

Reply:
What for? What should I do if it's null? If I don't return anything, it'll be null anyway just like it's now.